flow-typed/box-annotations.js

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
This seems to be duplicate from a separate PR? Can you clean up the patchset to include only what it needs?